src/build_test.cc

```diff
@@ -2258,47 +2258,6 @@ TEST_F(BuildTest, DepsGccWithEmptyDepfileErrorsOut) {
   ASSERT_EQ(1u, command_runner_.commands_ran_.size());
 }
 
-TEST_F(BuildTest, StatusFormatElapsed_e) {
```

Comment:
Good, but can you put this commit before the previous one, this will make the addition of `BuildTest.StatusFormatValidator` clearer.